src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/prometheus/silence-form/silence-form.component.html

```diff
@@ -1,218 +1,194 @@
-          </div>
-        </div>
-
-        <!-- Comment -->
-        <div class="form-group row">
-          <label class="cd-col-form-label required"
-                 for="comment"
-                 i18n>Comment</label>
-          <div class="cd-col-form-input">
-            <textarea class="form-control"
-                      formControlName="comment"
-                      id="comment"
-                      name="comment"
-                      type="text">
-            </textarea>
-            <span *ngIf="form.showError('comment', formDir, 'required')"
-                  class="invalid-feedback"
-                  i18n>This field is required!</span>
-          </div>
-        </div>
-
-        <!-- Start time -->
-        <div class="form-group row">
-          <label class="cd-col-form-label"
-                 for="starts-at">
-            <span class="required"
-                  i18n>Start time</span>
-            <cd-helper i18n>If the start time lies in the past the creation time will be used</cd-helper>
-          </label>
-          <div class="cd-col-form-input">
-            <input class="form-control"
-                   formControlName="startsAt"
-                   [ngbPopover]="popStart"
-                   triggers="manual"
-                   #ps="ngbPopover"
-                   (click)="ps.open()"
-                   (keypress)="ps.close()">
-            <span *ngIf="form.showError('startsAt', formDir, 'required')"
-                  class="invalid-feedback"
-                  i18n>This field is required!</span>
-          </div>
-        </div>
-
-        <!-- Duration -->
-        <div class="form-group row">
-          <label class="cd-col-form-label required"
-                 for="duration"
-                 i18n>Duration</label>
-          <div class="cd-col-form-input">
-            <input class="form-control"
-                   formControlName="duration"
-                   id="duration"
-                   name="duration"
-                   type="text">
-            <span *ngIf="form.showError('duration', formDir, 'required')"
-                  class="invalid-feedback"
-                  i18n>This field is required!</span>
-          </div>
-        </div>
-
-        <!-- End time -->
-        <div class="form-group row">
-          <label class="cd-col-form-label required"
-                 for="ends-at"
-                 i18n>End time</label>
-          <div class="cd-col-form-input">
-            <input class="form-control"
-                   formControlName="endsAt"
-                   [ngbPopover]="popEnd"
-                   triggers="manual"
-                   #pe="ngbPopover"
-                   (click)="pe.open()"
-                   (keypress)="pe.close()">
-            <span *ngIf="form.showError('endsAt', formDir, 'required')"
-                  class="invalid-feedback"
-                  i18n>This field is required!</span>
-          </div>
+  <div class="form-header">
```

Comment:
i see that most of the items in the form is required, and only one or 2 is optional. so let's not mark the required fields.. we just need to mark only the optional ones.